test/modules/linear_boltzmann_solvers/transport_adjoint/response_2d_3.lua

```diff
@@ -162,15 +162,15 @@ function ResponseFunction(xyz, mat_id)
 end
 response_func = opensn.LuaSpatialMaterialFunction.Create({ lua_function_name = "ResponseFunction" })
```

Comment:
[repeat from another comment] also the name SpatialMaterialFunction feels weird now. When doing the adjoint, the distributed/volumetric source was often associated with a material (the "detector cross section"). But now that distributed sources can be used in forward mode, I do not think that associating a source with a material is what we wanted to keep. It is now a bit awkward to me to read SpatialMaterialFunction here now.